src/ProofOfConcepts/ACLFilterChips/KafkaInstanceLabel.js

```diff
@@ -1,5 +1,6 @@
 import "./customStyles.css";
 import React from "react";
+import { Label } from "@patternfly/react-core";
```

Comment:
You need to move import "./customStyles.css"; after you import the Label in all the .js files.

Reply:
I applied this and removed the "!important" from "ResourceTypeLabel.css",, but "mas-m-solid" still doesn't seem to apply